Clarify granular permissions opt-in is irreversible for read-only users (#9694)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/manage-access/enterprise-permissions.md

- Includes `user_credential_write`, so users can view and edit their own user credentials on **Your profile** > **Credentials** without access to the <Constant name="studio_ide" /> or <Constant name="platform_cli" />. Read-only users still need personal user credentials on this page to run warehouse queries in analysis features such as <Constant name="insights" /> and the <Constant name="semantic_layer" />.
- No write access and no access to develop in the <Constant name="studio_ide" /> or <Constant name="platform_cli" />.

> To access the capabilities of this permission set, you _must_ add users to a group that's assigned the Analyst read permission set. Users won't have access until they're added to the group. For the setup steps, refer to [Set up read-only user access](/docs/platform/manage-access/about-user-access#set-up-read-only-user-access). If you have Read-Only users, note that [enabling granular permissions](/docs/platform/manage-access/about-user-access#enable-granular-permissions-for-read-only-users) for your account is a one-time, irreversible setting.

Author: ```suggestion
To access the capabilities of this permission set, you _must_ add users to a group that's assigned the Analyst read permission set. Users won't have access until they're added to the group. For the setup steps, refer to [Set up read-only user access](/docs/platform/manage-access/about-user-access#set-up-read-only-user-access). 

If you have Read-Only users, note that [you can enable granular permissions](/docs/platform/manage-access/about-user-access#enable-granular-permissions-for-read-only-users) for your account as a one-time, irreversible setting.
```

- Includes `user_credential_write`, so users can view and edit their own user credentials on **Your profile** > **Credentials** without access to the <Constant name="studio_ide" /> or <Constant name="platform_cli" />. Read-only users still need personal user credentials on this page to run warehouse queries in analysis features such as <Constant name="insights" /> and the <Constant name="semantic_layer" />.
- No write access and no access to develop in the <Constant name="studio_ide" /> or <Constant name="platform_cli" />.

> To access the capabilities of this permission set, you _must_ add users to a group that's assigned the Analyst read permission set. Users won't have access until they're added to the group. For the setup steps, refer to [Set up read-only user access](/docs/platform/manage-access/about-user-access#set-up-read-only-user-access). 

Reviewer: ```suggestion
To access the capabilities of this permission set, you _must_ add users to a group that's assigned the Analyst read permission set. Users won't have access until they're added to the group. For the setup steps, refer to [Set up read-only user access](/docs/platform/manage-access/about-user-access#set-up-read-only-user-access).
```

> To access the capabilities of this permission set, you _must_ add users to a group that's assigned the Analyst read permission set. Users won't have access until they're added to the group. For the setup steps, refer to [Set up read-only user access](/docs/platform/manage-access/about-user-access#set-up-read-only-user-access). 

Reviewer: Hiya Mirna

This new sentence about enabling granular permissions sits inside the Analyst read private beta expandable. That makes it easy to read granular permissions as part of Analyst read private beta, but the main write-up under Licenses applies to all Read-Only users, not only those on Analyst read.

Reviewer 2: Read-only is the license type and Analyst Read is the permission set. This can be a bit confusing so may be worth explicitly calling out "Read-Only license" and "Analyst Read permission set" in sentences

> If you have Read-Only users, note that [you can enable granular permissions](/docs/platform/manage-access/about-user-access#enable-granular-permissions-for-read-only-users) for your account as a one-time, irreversible setting.

Reviewer: ```suggestion
If you have Read-Only users, you can also [enable granular permissions](/docs/platform/manage-access/about-user-access#enable-granular-permissions-for-read-only-users) for your account. Enabling it is a one-time, irreversible setting.
```
